package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "monoplasma",
-  "version": "0.1.10",
+  "version": "0.2.1",
```

Comment:
I'd just bump to 0.1.11, since it's not breaking anything external

I'm saving 0.2 for the completely revamped architecture  ;)

Reply:
ok, I'll change and merge